www/app/docs/ai-embeddings/[[...slug]]/pg_vector/index.mdx

```diff
@@ -312,6 +312,52 @@ It is recommended to add indexes on the columns used for filtering, so the query
 Especially when the use of the filter can lead to small enough result sets that a sequential scan is faster than use of the vector index.
 This will optimize not just performance but also recall.
 
+With approximate indexes, filtering is applied after the index is scanned. If a condition matches 10% of rows,
+with HNSW and the default hnsw.ef_search of 40, only 4 rows will match on average.
+
+Starting in version `0.8.0`, you can enable **iterative index scans**, which will automatically scan more of the index when needed.
+
+### Iterative Index scans
+
+Using iterative index scans, Postgres will scan the approximate index for nearest neighbors, apply additional filters and, if the number of
+neighbors after filtering is insufficient, it will continue scanning until sufficient results are found.
+
+Each index has its own configuration (GUC) for iterative scans: `hnsw.iterative_scan` and `ivfflat.iterative_scan`.
+And these can be set to `strict_order` or `relaxed_order`.
```

Comment:
AFAICS only hnsw supports `strict_order`.  It would be good to document that `off` is also a valid value, and is the default value.